### DISCUSSION TITLE
How can I use on install omaga template in my proyect ?

### DISCUSSION THREAD

**Discussion by rrdopanga** [primefaces/community#192] [PrimeFaces Templates]
I have the basic licence to use omega template, but i don't know how I can use it in my own proyect and from where I can download the resources.

**Comment by danielemaddaluno** [comment]
@rrdopanga have you found any tutorial/video to integrate the template a JSF project?

**Comment by mertsincan** [comment]
Hi,

Duplicate of https://github.com/orgs/primefaces/discussions/742

Best Regards,
